lib/Minz/ExtensionManager.php

```diff
@@ -172,6 +172,14 @@ public static function load($info) {
 			                  '` is not an instance of `Minz_Extension`');
 			return null;
 		}
+		$reflector = new ReflectionClass($extension);
+		$className = $reflector->getName();
+		if ('Minz_Extension' === $reflector->getMethod('handleConfigureAction')->class ||
```

Comment:
I have not double-checked: Do we have a guarantee from before that the extension we are loading is inheriting from Minz_Extension? If not, this code will raise an exception if one of the methods does not exist

Reply:
Extension must be instance of Minz_Extension to be recognized by the manager. Thus we can be sure that the base class is Minz_Extension. So we always have at least the Minz_extension methods available. If the definition of those methods are in Minz_Extension, that means it was never overriden.